cmds/contest/server/server.go

```diff
@@ -170,10 +223,8 @@ func ServerMain(cmd string, args []string, sigs <-chan os.Signal) error {
 		log.Fatalf("Invalid target locker name %q", *flagTargetLocker)
 	}
 
-	plugins.Init(pluginRegistry, ctx.Logger())
```

Comment:
ok, this fixes loading plugins multiple times, but now we don't load the user functions at all anymore

Reply:
good point, I'll re-add user function loading